kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/IRestClient.java

```diff
@@ -21,15 +21,15 @@
 
 /** A underlying http client interface for common rest request. */
 public interface IRestClient extends AutoCloseable {
-  public <T> T get(String path, Map<String, Object> params, Class<T> type, String authHeader);
+  <T> T get(String path, Map<String, Object> params, Class<T> type, String authHeader);
```

Comment:
`public` is redundant here hence nothing other than `public` is allowed in `interface`